rules/art_prompts.yml

```diff
@@ -0,0 +1,234 @@
+
+
+drawing_prompt_character:
+# Comics
+- a hero
+- a villain
+- a supervillain
+
+# Magical creatures
+- a ghost
+- a monster
+- a troll
+- an elf
+- a wizard
+- a mermaid
+- an angel
+- a demon
+- a Jedi Knight
+- a Sith Lord
+- a Dalek
+
+# Other types of characters
+- a Pokémon
+- a robot
+- a cyborg
+- an alien
+- a muppet
```

Comment:
![](https://i.guim.co.uk/img/media/44856f0611a6ae5e8f4e1875772d8f462a7e6f10/0_139_1118_671/master/1118.jpg?width=620&quality=45&auto=format&fit=max&dpr=2&s=af389e4f827018ea16170e4db8eb4467)

Is this offensive?